track.c

```diff
@@ -250,6 +250,57 @@ void sorted_list_remove_track(struct list_head *head, struct rb_root *tree_root,
 	list_del(&track->node);
 }
 
+static void shuffle_array(void *array, size_t n, size_t size) 
+{
+	char tmp[size];
```

Comment:
Hm, is this the first place in cmus where var-sized arrays are used? I am not 100% sure I am comfortable with it for some reason.